src/lib/resource/sections/applicant-fatal-claim.js

```diff
@@ -115,10 +115,10 @@ module.exports = {
                 {
                     target: 'p-applicant-claim-type',
                     cond: ['|role.all', 'deceased']
-                },
+                },                
                 {
-                    target: 'p--context-applicant-details',
-                    cond: ['|role.all', 'nonDeceased']
+                    target: 'p-applicant-applied-before-for-this-crime',
+                    cond: ['|role.all', 'myself']
```

Comment:
I think this routing will break for someone who is applying for someone else non-deceased